tools/BatchModelGenerator/Program.cs

```diff
@@ -35,110 +35,13 @@ public class Program
-            {"Microsoft.Azure.Batch.OutputFileUploadOptions", "PSOutputFileUploadOptions"},
-            {"Microsoft.Azure.Batch.OutputFileBlobContainerDestination", "PSOutputFileBlobContainerDestination"},
-            {"Microsoft.Azure.Batch.PoolEndpointConfiguration", "PSPoolEndpointConfiguration"},
-            {"Microsoft.Azure.Batch.PoolInformation", "PSPoolInformation"},
-            {"Microsoft.Azure.Batch.PoolNodeCounts", "PSPoolNodeCounts"},
-            {"Microsoft.Azure.Batch.PoolSpecification", "PSPoolSpecification"},
-            {"Microsoft.Azure.Batch.PoolStatistics", "PSPoolStatistics"},
-            {"Microsoft.Azure.Batch.PoolUsageMetrics", "PSPoolUsageMetrics"},
-            {"Microsoft.Azure.Batch.PublicIPAddressConfiguration", "PSPublicIPAddressConfiguration"},
-            {"Microsoft.Azure.Batch.RecentJob", "PSRecentJob"},
-            {"Microsoft.Azure.Batch.ResizeError", "PSResizeError"},
-            {"Microsoft.Azure.Batch.ResourceFile", "PSResourceFile"},
-            {"Microsoft.Azure.Batch.ResourceStatistics", "PSResourceStatistics"},
-            {"Microsoft.Azure.Batch.Schedule", "PSSchedule"},
-            {"Microsoft.Azure.Batch.StartTask", "PSStartTask"},
-            {"Microsoft.Azure.Batch.StartTaskInformation", "PSStartTaskInformation"},
-            {"Microsoft.Azure.Batch.SubtaskInformation", "PSSubtaskInformation"},
-            {"Microsoft.Azure.Batch.TaskConstraints", "PSTaskConstraints"},
-            {"Microsoft.Azure.Batch.TaskContainerExecutionInformation", "PSTaskContainerExecutionInformation"},
-            {"Microsoft.Azure.Batch.TaskContainerSettings", "PSTaskContainerSettings"},
-            {"Microsoft.Azure.Batch.TaskCounts", "PSTaskCounts"},
-            {"Microsoft.Azure.Batch.TaskDependencies", "PSTaskDependencies"},
-            {"Microsoft.Azure.Batch.TaskExecutionInformation", "PSTaskExecutionInformation"},
-            {"Microsoft.Azure.Batch.TaskInformation", "PSTaskInformation"},
-            {"Microsoft.Azure.Batch.TaskIdRange", "PSTaskIdRange"},
-            {"Microsoft.Azure.Batch.TaskFailureInformation", "PSTaskFailureInformation"},
-            {"Microsoft.Azure.Batch.TaskSchedulingPolicy", "PSTaskSchedulingPolicy"},
-            {"Microsoft.Azure.Batch.TaskStatistics", "PSTaskStatistics"},
-            {"Microsoft.Azure.Batch.UploadBatchServiceLogsResult", "PSStartComputeNodeServiceLogUploadResult"},
-            {"Microsoft.Azure.Batch.UsageStatistics", "PSUsageStatistics"},
-            {"Microsoft.Azure.Batch.UserAccount", "PSUserAccount"},
-            {"Microsoft.Azure.Batch.UserIdentity", "PSUserIdentity"},
-            {"Microsoft.Azure.Batch.VirtualMachineConfiguration", "PSVirtualMachineConfiguration"},
-            {"Microsoft.Azure.Batch.WindowsConfiguration", "PSWindowsConfiguration"},
-            {"Microsoft.Azure.Batch.WindowsUserConfiguration", "PSWindowsUserConfiguration"},
+          { "Microsoft.Azure.Batch.NodeFile", "PSNodeFile" },
```

Comment:
Does this PSNodeFile need a custom mapping? Seems like it matches the default naming.

Reply:
Squashed my changes. Here's the file: https://github.com/Azure/azure-powershell/blob/eafed1824eb5c4ec1e2c1b57fdd823eab74b434e/tools/BatchModelGenerator/Program.cs